content/articles/how-does-a-dokku-work.md

```diff
@@ -0,0 +1,185 @@
+Title: How does a Heroku work?
+Date: 2015-01-12
+Short_summary: We explore how Heroku makes data from your Git repository appear as a web service.
+Category: Misc
```

Comment:
we should probably have proper categories rather than putting everything in misc

@teh @Gautier 
I'm thinking Dev, Infra, Business, Design, Misc